client/src/components/StagingWarningModal/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { Button, Modal } from '@freecodecamp/react-bootstrap';
+import { useTranslation } from 'react-i18next';
+
+import store from 'store';
+import { Spacer } from '../helpers';
+
+function StagingWarningModal(): JSX.Element {
+  const { t } = useTranslation();
+  const [show, setShow] = useState(
+    !((store.get('isStagingEnv') as boolean) === true)
+  );
```

Comment:
```suggestion
  const [show, setShow] = useState(!store.get('acceptedStagingWarning'));
```
There's no need to assert the type since `!` converts anything to a boolean. Also, we're not storing the fact that it's a staging environment - we want to know if they've accepted the warning.

Reply:
done